knowledge-base/how-to-display-a-report-via-instancereportsource-in-the-html5-viewer.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How to display a report via InstanceReportSource in the HTML5 Viewer
+description: How to display a report via InstanceReportSource in the HTML5 Viewer.
+type: how-to
+page_title: How to display a report via InstanceReportSource in the HTML5 Viewer
+slug: how-to-display-a-report-via-instancereportsource-in-the-html5-viewer
+res_type: kb
+---
+
+## Description
+
+How to display a report via InstanceReportSource in the HTML5 Viewer.  
```

Comment:
Add some more words on why you would want to use this approach. What benefits it gives and what drawbacks it has (if any). Put the reader in a concrete scenario where this is helpful.